main-app/app/tests/controller-test/main-controller-test.js

```diff
@@ -0,0 +1,45 @@
+(function () {
+    'use strict';
+    describe('Main controller tester', function () {
+        var $scope,
+            sandbox,
+            makingMovesSpy,
+            humangameplaySpy,
+            newGameMakerSpy,
+            controller;
+
+        beforeEach(function () {
+            module('Tombola.NoughtsandCrosses');
+            inject(function ($rootScope, $controller) {
+                $scope = $rootScope.$new();
+                sandbox = sinon.sandbox.create();
+                makingMovesSpy = sinon.sandbox.spy(mocks.game, 'makingMoves');
+                humangameplaySpy = sinon.sandbox.spy(mocks.game, 'humangameplay');
+                newGameMakerSpy = sinon.sandbox.spy(mocks.game, 'startNewGame');
+                controller = $controller('MainController', {
+                    $scope: $scope,
+                    Game: mocks.game
+                });
+            });
+        });
+
+        it('Makingmoves should have been called at least once', function(){
+            $scope.makingMoves();
+            makingMovesSpy.should.have.been.calledOnce; /*jshint ignore:line*/
```

Comment:
You can do this like this if you wish, but given we're forced to use calledOnce by the framework, I think it is  legit to set up specific rules for test linting:
````
unittests: {
     src: ['./main-app/tests/**/*.js'],
    options: {
         '-W030': true,
        "globals"   : {
            'describe'   : false,
            'it'         : false,
            'before'     : false,
            'beforeEach' : false,
            'after'      : false,
            'afterEach'  : false,
         }
  }
    },
````